lib/src/cli.rs

```diff
@@ -699,6 +708,19 @@ async fn container_import(
     Ok(())
 }
 
+/// Grouping of metadata about an object.
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct RawMeta {
+    /// When the image was created. Sync it with the io.container.image.created label.
+    pub created: Option<String>,
+    /// Top level labels, to be prefixed to the ones with --label
+    pub labels: Option<BTreeMap<String, String>>,
```

Comment:
I'm OK having these here, but it seems like it'd make more sense for them to be separate CLI arguments?

Reply:
I think its valuable to have 2 sources for this, as currently rechunk will also generate the labels and the created tag and harmonize them.  Since it pulls the rpm database it can do fancy stuff such as variable substitution. So this acts as a way of passing them through the 2 commands cleanly.

Having to refeed them into arguments would be hell.

I expect if anyone else tried to extend this they would agree.

Here is how the action example for the layers looks right now:
https://github.com/hhd-dev/rechunk/blob/496f4b84aced656b9c2c0f176f24323fe13129ad/.github/workflows/online_test_deck.yml#L51-L93

CLI may override the file.